components/globe-component/globe-component.js

```diff
@@ -277,6 +299,13 @@ export default {
       const dirLight = new THREE.DirectionalLight(0xffaa66, 4.2)
       dirLight.position.set(15, 13, 15)
 
+      // const that = this
+
+      // this.controls.addEventListener('change', function () {
+      //   console.log(this.object.position.z)
+      //   that.avatar.updateDistanceFromGlobe(this.object.position.z)
+      // }, false)
```

Comment:
You can remove this inactive code I guess?

Reply:
yes!